pdfminer/pdfdocument.py

```diff
@@ -485,27 +499,35 @@ def get_cfm(self, name):
 
     def authenticate(self, password):
         password = password.encode('utf-8')[:127]
-        hash = SHA256.new(password)
+        hash = sha256(password)
         hash.update(self.o_validation_salt)
         hash.update(self.u)
         if hash.digest() == self.o_hash:
-            hash = SHA256.new(password)
+            hash = sha256(password)
             hash.update(self.o_key_salt)
             hash.update(self.u)
-            return AES.new(hash.digest(), mode=AES.MODE_CBC, IV=b'\x00' * 16)\
-                .decrypt(self.oe)
-        hash = SHA256.new(password)
+            cipher = Cipher(algorithms.AES(hash.digest()),
+                            modes.CBC(b'\0' * 16),
+                            backend=default_backend())
+            return cipher.decryptor().update(self.oe)
+        hash = sha256(password)
         hash.update(self.u_validation_salt)
         if hash.digest() == self.u_hash:
-            hash = SHA256.new(password)
+            hash = sha256(password)
             hash.update(self.u_key_salt)
-            return AES.new(hash.digest(), mode=AES.MODE_CBC, IV=b'\x00' * 16)\
-                .decrypt(self.ue)
+            cipher = Cipher(algorithms.AES(hash.digest()),
+                            modes.CBC(b'\0' * 16),
+                            backend=default_backend())
+            return cipher.decryptor().update(self.ue)
         return None
 
     def decrypt_aes256(self, objid, genno, data):
-        return AES.new(self.key, mode=AES.MODE_CBC, IV=data[:16])\
-            .decrypt(data[16:])
+        iv = data[:16]
+        ct = data[16:]
```

Comment:
Giving these parts a name, but `iv` and `ct` don't really help to understand what it is. You should use a more verbose name.